src/odemis/acq/stitching/_tiledacq.py

```diff
@@ -267,22 +315,85 @@ def _getNumberOfTiles(self):
                      for s, f in zip(self._area_size, reliable_fov)]
         nx = math.ceil(area_size[0] / reliable_fov[0])
         ny = math.ceil(area_size[1] / reliable_fov[1])
-        logging.debug("Calculated number of tiles nx= %s, ny= %s" % (nx, ny))
 
         # We have a little bit more tiles than needed, we then have two choices
         # on how to spread them:
         # 1. Increase the total area acquired (and keep the overlap)
         # 2. Increase the overlap (and keep the total area)
         # We pick alternative 1 (no real reason)
-        center = (self._area[0] + self._area[2]) / 2, (self._area[1] + self._area[3]) / 2
+        xmin, ymin, xmax, ymax = self._polygon.bounds
+        center = ((xmin + xmax) / 2, (ymin + ymax) / 2)
         total_size = nx * reliable_fov[0], ny * reliable_fov[1]
```

Comment:
Not completely sure if it matters here, but the total size should include one overlap as well. Since for the total width you have one full tile and (n-1) overlapping tiles
```suggestion
        total_size = nx * reliable_fov[0] + self._sfov[0] * self._overlap, 
                     ny * reliable_fov[1] + self._sfov[1] * self._overlap
```

Reply:
I am also not sure here. @pieleric can you help?

Reply:
Indeed, Thera's remark sounds correct. For instance, with just 2 tiles, you'd have 2 full FoVs - 1 overlap. This correctly computed by Thera's formula.

Reply:
I have applied the suggested change. Given the change the starting position (center of top-left first tile) will change for all systems.